### PR TITLE
Monica finish

### PR DESCRIPTION
**Pull request by bustamonica**
Add more mock data.
Fix some bugs when viewing vehicles.
Enhanced UI.

### CODE REVIEW

**Comment by bustamonica** [comment]
changed the drop down to dynamically compiled

File: src/ca/ubc/cs304/controller/MakeReservationController.java [hunk 1]
```diff
@@ -21,7 +22,30 @@ public MakeReservationController(JFrame currentWindow) {
...
-        if (!checkDateIsValid(reservation.getFromDate(), reservation.getToDate())) {
+        System.out.println("fromDate:"+reservation.getToTime());
+
+        if (!checkDateIsValid(reservation.getFromDate(), reservation.getToDate(), reservation.getFromTime(), reservation.getToTime())) {
+            currentWindow.dispose();
+            ErrorWindow errorWindow = new ErrorWindow();
+            errorWindow.infoBox("Sorry, time information is invalid", "Invalid Input!");
+            MakeReservationWindow makeReservationWindow = new MakeReservationWindow();
+            MakeReservationController makeReservationController = new MakeReservationController(makeReservationWindow);
+            ArrayList<String> vtlist = dbHandler.findAllVT();
```
Comment (ayapcck) [inline src/ca/ubc/cs304/controller/MakeReservationController.java:33]: you can move these to the top , above the if statement, and won't have to repeat so much code

**Comment by taran1598** [comment]
I resolved the addTablesAndData.sql file

File: src/ca/ubc/cs304/ui/MakeReservationWindow.java [hunk 2]
```diff
@@ -54,15 +54,20 @@ public void showMenu(MakeReservationDelegate makeReservationDelegate, String lic
...
         if (this.license == null) {
             placeFieldAndLabel("License", licenseField, contentPane, gb, c);
         }
 
         // place car type
         JLabel carType = new JLabel("Car Type:");
         placeLabel(carType, contentPane, gb, c, 10, 5);
-        String[] vtList = {"SUV", "full-size", "truck", "economy", "mid-size", "standard"};
-        vtBox = new JComboBox(vtList);
+        String[] vtList1 = new String[vtList.size()];
```
Comment (ayapcck) [inline src/ca/ubc/cs304/ui/MakeReservationWindow.java:65]: I think you can convert directly to an String[] using 
`result.toArray(new String[result.size()])`

File: src/ca/ubc/cs304/ui/MakeReservationWindow.java [hunk 3]
```diff
@@ -73,7 +78,12 @@ public void showMenu(MakeReservationDelegate makeReservationDelegate, String lic
         // place location label and field
         JLabel location = new JLabel("Location:");
         placeLabel(location, contentPane, gb, c, 0, 10);
-        String[] locationList = {"shop_1", "shop_2", "shop_3", "shop_4"};
+        String[] locationList = new String[branchList.size()];
+        System.out.println(branchList.size());
+        for (int i = 0; i < branchList.size(); i++) {
```
Comment (ayapcck) [inline src/ca/ubc/cs304/ui/MakeReservationWindow.java:83]: same here

File: src/ca/ubc/cs304/ui/MakeReservationWindow.java [hunk 4]
```diff
@@ -85,8 +95,11 @@ public void showMenu(MakeReservationDelegate makeReservationDelegate, String lic
         // place city label and field
         JLabel cityLabel = new JLabel("City:");
         placeLabel(cityLabel, contentPane, gb, c, 0, 10);
-        String[] cityList = {"Vancouver", "Richmond", "Burnaby", "Coquitlam"};
-        cityBox = new JComboBox(cityList);
+        String[] cityList1 = new String[cityList.size()];
+        for (int i = 0; i < cityList.size(); i++) {
```
Comment (ayapcck) [inline src/ca/ubc/cs304/ui/MakeReservationWindow.java:99]: and here

**Review by ayapcck — CHANGES_REQUESTED** [review]

**Review by taran1598 — APPROVED** [review]

**Review by ayapcck — APPROVED** [review]

**Pull request merged**